device/src/main/java/com/aconno/sensorics/device/format/FormatLocatorUseCaseImpl.kt

```diff
@@ -0,0 +1,16 @@
+package com.aconno.sensorics.device.format
+
+import com.aconno.sensorics.domain.format.AdvertisementFormat
+import com.aconno.sensorics.domain.format.FormatLocatorUseCase
+import com.aconno.sensorics.domain.repository.FormatRepository
+import io.reactivex.Single
+
+class FormatLocatorUseCaseImpl(
+    private val formatRepository: FormatRepository
+) : FormatLocatorUseCase {
```

Comment:
Check if we are still using this use case

Reply:
Removed.